Source/Extensions/Blazorise.Components/RouterTabsService.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazorise.Components;
+
+internal class RouterTabsItem
+{
+    public string Name { get; set; }
+    public string TypeName { get; set; }
+    public string Url { get; set; }
+    public RenderFragment Body { get; set; }
+    public string TabCssClass { get; set; }
+    public string TabPanelCssClass { get; set; }
+    public bool Closeable { get; set; } = true;
+
+}
+
+public class RouterTabsService
+{
+
+    internal event Action OnStateHasChanged;
```

Comment:
We usually don't name events with `On` prefixes. The handlers have it usually.

Reply:
`StateHasChangedHandler`?

Reply:
`StateHasChanged`.